kotlin-analysis-gradle/src/main/kotlin/org/jetbrains/research/ml/kotlinAnalysis/gradle/GradleFileUtil.kt

```diff
@@ -6,7 +6,7 @@ package org.jetbrains.research.ml.kotlinAnalysis.gradle
 class GradleFileUtil {
     companion object {
 
-        private const val NAME = "[^:\'\"]*"
+        private const val NAME = "[^:{}\'\"]*"
         private const val QUOTES = "[\'\"]"
         private const val SEPARATORS = "[\'\":,]+"
 
```

Comment:
Please, add some description with examples to GRADLE_DEPENDENCIES_SHORT_REGEX, GRADLE_DEPENDENCIES_FULL_REGEX and GRADLE_DEPENDENCIES_KOTLIN_REGEX and why we have a separated one for kotlin)